src/main/scala-2.12/com/typesafe/sbt/packager/PluginCompat.scala

```diff
@@ -0,0 +1,46 @@
+package com.typesafe.sbt.packager
+
+import java.nio.file.{Path => NioPath}
+import java.util.jar.Attributes
+import sbt.*
+import xsbti.FileConverter
+
+object PluginCompat {
```

Comment:
this is no longer private. should we consider it part of this plugin public API? Or somehow move it to an internal package (like `com.typesafe.sbt.packager.internal`) to signal it should not be used?

Reply:
Maybe the type aliases needs to be public since they are part of keys, but individual functions should be scoped to `private[packager]`?

Reply:
Added `private[packager]` in 2a20c607c0b476feff5d959f06983d243e6f0922